spec/helpers/hyrax_helper_spec.rb

```diff
@@ -82,7 +82,6 @@ def new_state
           car_link   = search_catalog_path(f: { 'vehicle_type_sim' => ['car'] })
           truck_link = search_catalog_path(f: { 'vehicle_type_sim' => ['truck'] })
           expect(subject).to eq "<a href=\"#{car_link}\">car</a>, <a href=\"#{truck_link}\">truck</a>"
-          expect(subject).to be_a ActiveSupport::SafeBuffer
```

Comment:
If we no longer expect this to return an `ActiveSupport::SafeBuffer`, should we also reflect this in the YARD doc for the `#link_to_facet_list` method above?

Reply:
Yes, I wasn't sure what it should return. Ideas?

Reply:
@jcoyne Is it just returning a string now?

Reply:
Sometimes a string and sometimes a SafeBuffer.

Reply:
Should we make the method return a SafeBuffer consistently?

Reply:
No. It's perfectly reasonable to return a string that isn't safe... it will just be escaped...which is good because that prevents XHR attacks.

Reply:
Works for me.